docs/UserGuide.md

```diff
@@ -3,8 +3,11 @@ layout: page
 title: User Guide
 ---
 
-Tutor's Pet is a **desktop application for managing students and classes, optimised for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI).
-If you can type fast, Tutor's pet can get your class management tasks done faster than traditional GUI apps.
+As a Computer Science teaching assistant, the traditional approach of managing students within your class would be to manually record their progress and attendance.
+However, as you will notice after some time, manual students tracking is unnecessarily time-consuming. This is why we have created Tutor's Pet.
+
+Tutor’s Pet is a desktop application that helps Computer Science teaching assistants manage their students. 
+It allows teaching assistants to record the classes that they teach, add students into those classes, and keep track of students’ attendance in those lessons.
```

Comment:
The phrasing of this sentence is rather weird, especially the `add students into those classes` part. I have an alternative below, but open to better phrasing as well.

```suggestion
It allows teaching assistants to keep track of the students in their classes, and record their attendance and participation scores.```